array-methods.js

```diff
@@ -88,7 +137,14 @@ var higherStateSums = null;
   if true set `areStatesInHigherStateSum` to `true`
   otherwise set it to `false`
  */
-var areStatesInHigherStateSum = null;
+const areStatesInHigherStateSum = Object.entries(stateSums)
+  .filter(function (arr) {
+    return ["WI", "IL", "WY", "OH", "GA", "DE"].includes(arr[0]);
+  }).every(function (elem) {
+    if (elem[1] > 2550000) {
```

Comment:
You could just return the result of the if conditional since that will evaluate to true/false anyway.